client/src/app/tabs/cloud-bpmn/BpmnEditor.js

```diff
@@ -463,6 +463,8 @@ export class BpmnEditor extends CachedComponent {
         }
       });
 
+      this.getModeler().get('linting').activate();
```

Comment:
I don't understand this duplication.

Why don't we react to a `linting.activated` and trigger the `onLayoutChanged` from there? Overall there seems to be duplication going on here.